src/components/claim/ClaimRewardsVeDF.svelte

```diff
@@ -75,112 +76,123 @@
   }
 </script>
 
-<div class={`container`}>
-  <h2 class="title">Earn OCEAN Rewards</h2>
-  <ClaimItem
-    title="Passive"
-    description="<p>Earn Passive Rewards from Data Farming by <strong>locking OCEAN</strong> and <strong>holding veOCEAN</strong>.</p><p>Current round passive rewards are distributed for <strong>last round</strong> holding.</p>"
-    distributedAmount={roundInfo?.passive}
-    apy={`${
-      $APYs
-        ? $APYs?.passive > 10000
-          ? "over 10000"
-          : `${$APYs?.passive.toFixed(2)}`
-        : parseFloat(0).toFixed(2)
-    }% Avg APY ${
-      $userAddress
-        ? `| ${
-            $APYs
-              ? $APYs?.passiveUser > 10000
-                ? "over 10000"
-                : `${$APYs?.passiveUser.toFixed(2)}`
-              : parseFloat(0).toFixed(2)
-          }% Your APY`
-        : ""
-    }`}
-    apyTooltip={descriptions.default.tooltip_rewards_apy_passive}
-    showRedirectLink={!$oceanUnlockDate && $veClaimables <= 0}
-    redirectLink={{ text: "Get veOCEAN", url: "veocean" }}
-    amount={`${parseFloat($veClaimables).toFixed(2)} OCEAN`}
-    metrics={[
-      {
-        name: "balance",
-        value: `${
-          $userBalances[
-            getAddressByChainIdKey(process.env.VE_SUPPORTED_CHAINID, "veOCEAN")
-          ]
-            ? parseFloat(
-                $userBalances[
-                  getAddressByChainIdKey(
-                    process.env.VE_SUPPORTED_CHAINID,
-                    "veOCEAN"
-                  )
-                ]
-              ).toFixed(3)
+<div class="container">
+  <Card title="Reward Programs">
+    <p>
+      Data Farming consists of two reward programs. 
+      Each reward program is unique and offers different ways for 
+      participants to get engaged. To access Reward Programs, 
+      participants are required to hold veOCEAN.
+    </p>
+    <div class="rewardsContainer">
+      <ClaimItem
+        title="Passive"
+        description="<p>Earn Passive Rewards from Data Farming by <strong>locking OCEAN</strong> and <strong>holding veOCEAN</strong>.</p><p><strong>Earn more Rewards</strong> by locking more OCEAN or increasing your unlock time.</p>"
```

Comment:
```suggestion
        description="<p>Earn passive rewards from Data Farming by <strong>locking OCEAN</strong> and <strong>holding veOCEAN</strong>.</p><p><strong>Earn more rewards</strong> by locking more OCEAN or increasing your unlock time.</p>"
```

Reply:
We have capitalized these as Nouns because this is what the reward programs are called.